### ISSUE TITLE
Glyphs

### ISSUE THREAD

**Issue by mtennekes** [zonebuilders/zonebuilder#19]
I didn't realize until today that our ClockBoard zones make excellent glyphs:

![Dutch_city_population](https://user-images.githubusercontent.com/2444081/75037713-b5d58980-54ac-11ea-99f4-62bd6f2bc3b3.png)

I placed the script in the sandbox folder. Some code chunks can be improved and used for standalone functions. Most importantly, a function to overlay two tessellations, and recalculate statistics (counts or densities). `stplanr::od_aggregate` does similar thing, but uses centroids.

**Comment by Robinlovelace** [comment]
Indeed, nice-looking summary glyphs representing the complexity of a city with a small visual is one of the big +s of the zonebuilder approach it seems. Great to see!
